README.md

```diff
@@ -1,6 +1,46 @@
 # Bike Sharing Sample Application
 
-## Docker-Compose usage
+## Azure Dev Spaces
+
+1. Create an AKS cluster with Azure Dev Spaces by running the following script, specifying the AKS name and region:
+    
+    ```
+    source ./create-aks.sh bikesharing01 eastus2
```

Comment:
nit: `source` command should be unnecessary because there aren't any environment variables you're trying to export in `create-aks.sh`